src/js/common/schemaform/ObjectField.jsx

```diff
@@ -210,7 +221,7 @@ class ObjectField extends React.Component {
             {DescriptionField && <DescriptionField formData={formData} formContext={formContext} options={uiSchema['ui:options']}/>}
             {!textDescription && !DescriptionField && description}
           </div>}
-          {uiOptions.showPrefillMessage && formContext.prefilled && <PrefillMessage/>}
+          {this.showPrefill(uiOptions, formContext) && <PrefillMessage message={prefillMessage}/>}
```

Comment:
It's generally recommended that you return true or false (not truthy or falsy) values when using the `{condition && ..}` pattern in React.

Reply:
Thanks @jbalboni, `showPrefill` now returns booleans.